apps/emqx_connector/src/mqtt/emqx_connector_mqtt_schema.erl

```diff
@@ -42,17 +42,17 @@ fields("config") ->
         [
             {"ingress",
                 mk(
-                    hoconsc:union([none, ref(?MODULE, "ingress")]),
+                    ref(?MODULE, "ingress"),
                     #{
-                        default => undefined,
+                        required => {false, recursively},
                         desc => ?DESC("ingress_desc")
                     }
                 )},
             {"egress",
                 mk(
-                    hoconsc:union([none, ref(?MODULE, "egress")]),
+                    ref(?MODULE, "egress"),
                     #{
-                        default => undefined,
+                        required => {false, recursively},
```

Comment:
Q: could this conflict with [`egress.remote`](https://github.com/emqx/emqx/pull/9463/files#diff-decf9bef907b097d2f556e82ad1e175a479b99500147da709d560d39d5bb83e0R227) being required?

Reply:
no it doesn't.